### PR TITLE
Extend tune_relay_x86 tutorial to measure default and kernel level tune

### PR DESCRIPTION
**Pull request by elvin-n**
Currently tune_relay_x86 is slightly inconsistent compare to other examples in tutorials/autotvm directory. If we want to show different API how to deal with AutoTVM (kernel level tune vs graph level tune) and performance difference - here it is

### CODE REVIEW

File: tutorials/autotvm/tune_relay_x86.py [hunk 1]
```diff
@@ -256,6 +274,29 @@ def tune_and_evaluate(tuning_opt):
...
+#
+#    Evaluation of the network been tuned on kernel level:
+#    Compile...
+#    Evaluate inference time cost...
+#    Mean inference time (std dev): 3.2 ms (0.03 ms)
+#
+#    Evaluation of the network been tuned on graph level:
 #    Compile...
+#    Config for target=llvm -keys=cpu -link-params=0, workload=('dense_nopack.x86', ('TENSOR', (1, 512), 'float32'), ('TENSOR', (1000, 512), 'float32'), None, 'float32') is missing in ApplyGraphBest context. A fallback configuration is used, which may bring great performance regression.
+#    Config for target=llvm -keys=cpu -link-params=0, workload=('dense_pack.x86', ('TENSOR', (1, 512), 'float32'), ('TENSOR', (1000, 512), 'float32'), None, 'float32') is missing in ApplyGraphBest context. A fallback configuration is used, which may bring great performance regression.
```
Comment (comaniac) [inline tutorials/autotvm/tune_relay_x86.py:300]: Do we know the root cause of these warnings?

**Review by comaniac — APPROVED** [review]
LGTM. Thanks for the fix.

File: tutorials/autotvm/tune_relay_x86.py (hunk 1 above)
Comment (elvin-n) [reply tutorials/autotvm/tune_relay_x86.py:300]: Good question. These warnings appear after graph level tuning. I can only suppose that graph optimizer decided to change layout that led to selection of different kernels which were not optimized on the previous stage

**Comment by comaniac** [comment]
Thanks @elvin-n

**Pull request merged**